pkg/image/disk.go

```diff
@@ -80,6 +80,17 @@ func (img *DiskImage) InstantiateManifest(m *manifest.Manifest,
 		qcow2Pipeline := manifest.NewQCOW2(buildPipeline, rawImagePipeline)
 		qcow2Pipeline.Compat = img.Platform.GetQCOW2Compat()
 		imagePipeline = qcow2Pipeline
+	case platform.FORMAT_VAGRANT_LIBVIRT:
+		qcow2Pipeline := manifest.NewQCOW2(buildPipeline, rawImagePipeline)
+		qcow2Pipeline.Compat = img.Platform.GetQCOW2Compat()
+
+		vagrantPipeline := manifest.NewVagrant(buildPipeline, qcow2Pipeline)
```

Comment:
Wouldn't it make sense to explicitly pass the backend to `manifest.NewVagrant()` since the format is `VAGRANT_LIBVIRT`?

Reply:
That is done in #1556 (the followup to this PR) which actually adds a second provider.